src/lib/offline-api/validator/schema/field-validations-schema.ts

```diff
@@ -18,7 +18,7 @@ const rangeValidation = validation('range', range('number'))
 
 const regexp = validation('regexp', Joi.object({
   pattern: Joi.string(),
-  flags: Joi.string()
+  flags: Joi.string().allow(null)
```

Comment:
I'm wondering if we should also add `.optional()`. 
It seems like the API also accepts `flags` to be left out.

Reply:
Happy to add that if desired. When I initially wrote the test, I simply left off the `flags` key from the validation object and the test passed, so it seems like it was effectively optional. Only after I explicitly set the value to `null` did the test fail. The test is a little contrived, since it's unlikely anyone would try to explicitly set a null value on `flags`, and the validation was failing against fields on the entry returned from the API.

Reply:
Also, travis is failing because
```
Error: Please define all necessary "CONTENTFUL_INTEGRATION_*" environment variables
```
I'm a little confused, since that seems unrelated to this PR. Please advise if I need to be doing something differently here.

Reply:
That's happening because the integration tests need access tokens defined, and for security reasons travis will not expose access tokens to builds triggered by forks of the main repo.  
Otherwise anyone could make a PR that just logs the environment variables and the secrets are leaked. 
That does make things very annoying for anyone proposing a PR though, so we'll eventually have to look into a better solution. For now, don't worry about it.